core/datainput.gms

```diff
@@ -130,14 +130,62 @@ pm_shGasLiq_fe_up(ttot,regi,sector)=0;
 pm_shGasLiq_fe_lo(ttot,regi,sector)=0;
 
 
+*******************************************************************************
+**************     Technology data input read-in and manipulation       *******
+*******************************************************************************
+*** Note: future to be its own module perhaps
+*** Note: in module 5 there are more cost manipulation after initial capacities are calculated, be aware those can overwrite your technology values for policy runs if you set them here in the core
 ***---------------------------------------------------------------------------
-*** Import and set global data
+*** Reading in and initializing global data
 ***---------------------------------------------------------------------------
-table fm_dataglob(char,all_te)  "energy technology characteristics: investment costs, O&M costs, efficiency, learning rates ..."
+
+table fm_dataglob(char,all_te)  "Energy and CDR technology characteristics: investment costs, O&M costs, efficiency, learning rates ..."
 $include "./core/input/generisdata_tech.prn"
 $include "./core/input/generisdata_trade.prn"
 ;
 
+*** CG warning: some of the SSP1 and SSP5 costs are not consistent with the story line (e.g. under SSP1 blue H2 and some fossil fuel CCS technologies have lower costs than in SSP2). This is to be fixed in the future when new SSP storylines are implemented, unclear when (29-1-2024). In the future, SSP1 and SSP5 data should be implemented as switches to avoid errors
+*JH* SSP energy technology scenario
+table f_dataglob_SSP1(char,all_te)        "Techno-economic assumptions consistent with SSP1"
+$include "./core/input/generisdata_tech_SSP1.prn"
+;
+table f_dataglob_SSP5(char,all_te)        "Techno-economic assumptions consistent with SSP5"
+$include "./core/input/generisdata_tech_SSP5.prn"
+;
+
+$IFTHEN.WindOff %cm_wind_offshore% == "1"
+
+*CG* set wind offshore, storage and grid to be the same as wind onshore (later should be integrated into input data)
+* main difference between onshore and offshore is the difference in f32_factorStorage
+fm_dataglob(char,"storwindoff") = fm_dataglob(char,"storwind");
+fm_dataglob(char,"gridwindoff") = fm_dataglob(char,"gridwind");
+$ENDIF.WindOff
+
+*RP* include global flexibility parameters
+$include "./core/input/generisdata_flexibility.prn"
+$IFTHEN.WindOff %cm_wind_offshore% == "1"
+fm_dataglob("flexibility","storwindoff")  = 1.93;
+fm_dataglob("flexibility","windoff")  = -1;
+$ENDIF.WindOff
+***---------------------------------------------------------------------------
+*** Reading in and initializing regional data
+***---------------------------------------------------------------------------
+parameter p_inco0(ttot,all_regi,all_te)     "regionalized technology costs Unit: USD$/KW"
+/
+$ondelim
+$include "./core/input/p_inco0.cs4r"
+$offdelim
+/
+;
+
+*** initializing energy service capital
+pm_esCapCost(tall,all_regi,all_teEs) = 0;
```

Comment:
does energy service capital still exist in the latest REMIND version, after transport|complex and buildings|putty were deleted?

Reply:
there are sill some 
![image](https://github.com/remindmodel/remind/assets/20283904/fabc8c25-1e85-42d4-9e44-b3172025b287)
@robertpietzcker